misc/userscripts/cast

```diff
@@ -133,17 +133,23 @@ echo "jseval -q $(printjs)" >> "$QUTE_FIFO"
 
 tmpdir=$(mktemp -d)
 file_to_cast=${tmpdir}/qutecast
+program_=$(which castnow)
+
+if [[ "${program_}" == "" ]]; then
+    msg error "castnow can't be found..."
+    exit 1
+fi
 
 # kill any running instance of castnow
-pkill -f /usr/bin/castnow
+pkill -f ${program_}
```

Comment:
I think it's a good idea to quote these variable expansions.

Reply:
Yes!